utils/s3/client.cc

```diff
@@ -1056,7 +1100,7 @@ class client::do_upload_file : private multipart_upload {
             req._headers["x-amz-tagging"] = seastar::format("{}={}", _tag->key, _tag->value);
         }
         req.write_body("bin", len, [f = std::move(f)] (output_stream<char>&& out_) mutable {
```

Comment:
nit: lambda is no longer `mutable`

Reply:
Done

Reply:
Thanks. Consider making this two-line change a separate patch. This "add retries to client" is already bloated enough